tests/e2e/signin.spec.js

```diff
@@ -17,43 +16,7 @@ module.exports = {
     await browser.waitForElementNotPresent(".session", 10000, true)
     openMenu(browser)
     await browser.waitForElementVisible("#sign-out", 10000, true)
-  },
-  */
-  "Create local account": async function(browser) {

```

Comment:
Why did you remove this?

Reply:
There is no "Sign in with local account" even with insecure flag. Don't know why